test/Microsoft.Azure.WebJobs.Host.EndToEndTests/Scale/ScaleHostEndToEndTests.cs

```diff
@@ -88,58 +83,35 @@ public async Task ScaleManager_GetScaleStatusAsync_ReturnsExpected(bool tbsEnabl
             hostBuilder.ConfigureAppConfiguration((hostBuilderContext, config) =>
             {
                 // Adding host.json here
-                config.AddJsonStream(new MemoryStream(Encoding.ASCII.GetBytes(hostJson)));
+                config.AddJsonStream(new MemoryStream(Encoding.UTF8.GetBytes(hostJson)));
 
                 // Adding app settings
-                config.AddInMemoryCollection(new List<KeyValuePair<string, string>>()
+                config.AddInMemoryCollection(new Dictionary<string, string>
                 {
-                    new KeyValuePair<string, string>("app_setting1", "value1"),
-                    new KeyValuePair<string, string>("app_setting2", "value2")
+                    { "app_setting1", "value1" },
+                    { "app_setting2", "value2"},
+                    { "FeatureManagement:Microsoft.Azure.WebJobs.Host.EndToEndTests", "true" }
```

Comment:
Here's how a feature flag can be added. We can also control the root "FeatureManagement" name - we could have it be "WebJobsFeatures" or something.

Reply:
The root name should be "HostingConfig" as function host already use it.

Reply:
We don't want a name like "HostingConfig" in the SDK layer. We should use a name that makes sense from the WebJobs SDK point of view. Functions host can override the source as needed when registering its provider.